src/lib/MapCards.svelte

```diff
@@ -32,53 +32,55 @@
 <div class="cards max-cols-3">
   {#if maps.length !== 0}
     {#each maps as map}
-      <div class="card">
-        <a
-          href={`${import.meta.env.VITE_BEATSAVER_BASE || 'https://beatsaver.com'}/maps/${map.id}`}
-          class="image-link"
-        >
-          <img
-            src={`${import.meta.env.VITE_BEATSAVER_CDN_BASE || 'https://cdn.beatsaver.com'}/${
-              map.versions[0].hash
-            }.jpg`}
-            alt={map.name}
-          />
-        </a>
-        <div class="content">
-          <div>
-            <a
-              href={`${import.meta.env.VITE_BEATSAVER_BASE || 'https://beatsaver.com'}/maps/${
-                map.id
-              }`}
-              class="title"
-              title={map.name}
-            >
-              {map.name}
-            </a>
-            <Uploader uploader={map.uploader} curator={map.curator} />
-          </div>
-          <div class="tag-row-container">
-            <Tags tags={map.tags}></Tags><div class="zip-download-button-container"><ZipDownloadButton downloadURL={map.versions[0].downloadURL}/></div>
-          </div>
-          <div class="last-row-container">
-            <Difficulties
-              diffs={map.versions[0].diffs}></Difficulties><OneClickDownloadButton mapId={map.id + ''}
-              />
+      <div class="card-wrapper">
+        <div class="card">
+          <a
+            href={`${import.meta.env.VITE_BEATSAVER_BASE || 'https://beatsaver.com'}/maps/${map.id}`}
+            class="image-link"
+          >
+            <img
+              src={`${import.meta.env.VITE_BEATSAVER_CDN_BASE || 'https://cdn.beatsaver.com'}/${
```

Comment:
Not in scope of this PR, but it's a bit odd, that we construct the cover URL link ourselves instead of just using the one provided by the API. The latter seems more reliable.

Reply:
Agreed, might be a leftover from early development. Definitely not something to address in this PR though.